toolchain/docs/check.md

```diff
@@ -96,10 +92,21 @@ instruction, as described in
 -   Up to two additional, kind-specific members. For example `SemIR::Assign` has
     members `InstId lhs_id` and `InstId rhs_id`.
 
+-   Optionally, an `ElementIndex index` field for instructions that refer to a
```

Comment:
This is one of the kind-specific members described in the previous bullet, rather than a distinct special case. I don't think it's worth calling out as a special case here, but maybe the previous bullet could mention there are various different valid types for the kind-specific members?